domain/learningandworkprogress/src/main/kotlin/uk/gov/justice/digital/hmpps/domain/learningandworkprogress/induction/InductionSchedule.kt

```diff
@@ -37,4 +37,7 @@ enum class InductionScheduleStatus(val inScope: Boolean) {
   EXEMPT_PRISON_OPERATION_OR_SECURITY_ISSUE(false),
   EXEMPT_SECURITY_ISSUE_RISK_TO_STAFF(false),
   EXEMPT_SYSTEM_TECHNICAL_ISSUE(false),
+  EXEMPT_PRISONER_TRANSFER(false),
```

Comment:
Not really related to Reviews (but is related to KPIs in general) ... but whilst defining the swagger spec for getting Reviews/Review Schedule and thinking about the statuses of the Review Schedule, I realised that a Review being exempt due to prisoner transfer/release/death were all also applicable to the InductionSchedule (that we provisionally created a few weeks back)
Hence the change to this enum (in the domain), the enum in the entity, and the mapper class 👇